stdlib/dynArray.mli

```diff
@@ -0,0 +1,52 @@
+(** Dynamic Arrays.
+
+   This module implements amortized time complexity dynamic arrays.
+ *)
+
+type 'a dynArray
+(** The type of dynamic arrays containing elements of type ['a]. *)
+
+exception OutOfBounds
+(** Raised when an index is out of the array. *)
+
+val create : unit -> 'a dynArray
+(** Returns a new empty dynamic array.*)
```

Comment:
Given a possible confusion with `Array.create`, maybe using `empty` instead?

Reply:
That makes sense.
Given that wouldn’t there also be a confision with Array.make ?

Reply:
`Array.make` and `DynArray.make` have similar signature and behavior. It is a good thing they share a name.

Reply:
`empty` is for values. `create`/`make` are for creation functions. I don't see the confusion with `Array.create`. There is a create function is Hashtbl, Queue, etc...